src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/TemplateBuilder.cs

```diff
@@ -108,6 +96,40 @@ public IHtmlContent Build()
             // though _model may have been reset to null. Otherwise we might lose track of the model type /property.
             viewData.ModelExplorer = _modelExplorer.GetExplorerForModel(_model);
 
+            var formattedModelValue = viewData.Model;
```

Comment:
Wait, why `viewData.Model`? Just updated that value 2 lines above and it's more obvious to use `_model`.